frontend/cypress/integration/dataSources.spec.js

```diff
@@ -22,6 +22,40 @@ describe('The Data Sources Page', () => {
     cy.get('.sources-table').find('tbody').find('tr').should('have.length.lessThan', 11);
   });
 
+  it('shows correct url on pagination of data sources', () => {
```

Comment:
This could be more general to cover all expected behavior on pagination ... i.e.
`it('paginates when datasets exceed 10')`

Reply:
I'd expect to include a check on the URL and the results to make sure it actually paginated. Ofc you'd need to first verify that this ONLY happens when results exceed i.e. you'll need to render a table with less than 10 and verify that there's no pagination, then another with more than 10 to verify the reverse.